roadmap.md

```diff
@@ -97,6 +95,10 @@ Microsoft Distro
 
 Microsoft ships multiple .NET Core distros. It is important that Microsoft can successfully ship .NET Core at quality and meet its desired dates.
 
+Other Distros
+================
+.NET Core will ship as part of many Linux distros and we are actively working with key partners in the Linux community to make it natural for .NET Core to get everyone people need it. We are contantly looking to expand our distro support and welcome contributions and collaborations in this direction.
```

Comment:
> to make it natural for .NET Core to get everyone people need it

I don't understand this part. Was it supposed to be "everywhere people need it"?